packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -812,5 +801,20 @@ + (NSString *)escapedResourceName:(NSString *)name
   return [name stringByAddingPercentEncodingWithAllowedCharacters:allowedCharacters];
 }
 
++ (NSString *)documentDirectoryForExperienceId:(NSString *)experienceId
```

Comment:
How about moving those methods to instance methods of `EXFileSystemService` so we can just override them in `EXFileSystemBinding` and not override `EXFileSystem` at all? 🤔